src/views/Home/index.view.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from "react";
 import Hero from "views/Home/components/Hero/index.view";
+import Footer from "views/Home/components/Footer/index.view";
```

Comment:
One last thing, we don't need to import all the way up the tree from `views/` directory. We can simply import using `./components/Footer/index.view`

Reply:
Cool ~ I would recommend changing this in the previous line too in another pull request.

Reply:
Since @mocklee 's PR (#122) deals with `Hero` component, I will leave a comment there to change line 2.

Reply:
Sounds good!